lib/Mandrill/Mandrill.php

```diff
@@ -208,4 +227,45 @@ public function log($msg)
             error_log($msg);
         }
     }
+
+    /**
+     * @return array
+     */
+    protected function getCurlOptions()
+    {
+        return $this->_curlOptions;
+    }
+
+    /**
+     * @param array $curlOptions
+     */
+    protected function setCurlOptions($curlOptions)
+    {
+        $this->_curlOptions = $curlOptions;
+    }
+
+    /**
+     * @param $index
+     * @param $value
+     */
+    protected function setCurlOptionsIndexValue($index, $value)
+    {
+        $this->_curlOptions[$index] = $value;
+    }
+
+    /**
+     * @param array $curlOptions
+     */
+    protected function setCurlOptionsAddOptions($curlOptions = array())
+    {
+        $this->_curlOptions += $curlOptions;
+    }
```

Comment:
What is the purpose of these functions?

Reply:
They was intended for being used if needed. Now they are removed.